### PR TITLE
[mqtt] Add default translations

### LINKED ISSUE #12220: [mqtt.generic] There is not yet any default translation properties file.
There is not yet any default translation properties file.

_Originally posted by @lolodomo in https://github.com/openhab/openhab-addons/issues/12167#issuecomment-1030785190_

I tried creating the translations file for `org.openhab.binding.mqtt.generic` using this command:
```
$ mvn org.openhab.core.tools:i18n-maven-plugin:3.2.0:generate-default-translations -pl :org.openhab.binding.mqtt.generic
```

It created a file called `mqtt.properties`

Questions:
1. Should the existing mqttbroker_de.properties be deleted? I believe this file is an orphaned file from an older binding structure
2. The `mqtt.properties` file that was generated from the command above, don't include the texts from `thing/channels.xml` or `thing/generic.thing.xml` -  @wborn is this perhaps a bug in the `generate-default-translations` maven plugin?

### PR DESCRIPTION
**Pull request by wborn**
This will allow for translating the MQTT Binding in Crowdin.

Fixes #12220

### CODE REVIEW

File: bundles/org.openhab.binding.mqtt.espmilighthub/src/main/resources/OH-INF/i18n/mqtt.properties [hunk 1]
```diff
@@ -1,3 +1,20 @@
+# thing types
+
+thing-type.mqtt.cct.label = Milight CCT
```
Comment (martinvw) [inline bundles/org.openhab.binding.mqtt.espmilighthub/src/main/resources/OH-INF/i18n/mqtt.properties:3]: I'm a bit confused about this line in this module but if that is fine LGTM

**Review by martinvw — APPROVED** [review]

File: bundles/org.openhab.binding.mqtt.espmilighthub/src/main/resources/OH-INF/i18n/mqtt.properties (hunk 1 above)
Comment (wborn) [reply bundles/org.openhab.binding.mqtt.espmilighthub/src/main/resources/OH-INF/i18n/mqtt.properties:3]: The MQTT Binding also has a bundle for integrating an open source based Milight hub using MQTT (see [add-on docs](https://www.openhab.org/addons/bindings/mqtt.espmilighthub/#espmilighthub-binding)).

This is what all MQTT Binding things look like in the UI :slightly_smiling_face: 


Screenshot from 2022-02-20 16-55-32

**Comment by lolodomo** [comment]
Just one question: due to the lack of files binding.xml, there is no bindings label/description to internationalize. Is it expected ?

**Comment by wborn** [comment]
Yes it's one binding consisting of several bundles. The binding.xml file is already translated in the org.openhab.binding.mqtt bundle.

**Comment by lolodomo** [comment]
Ok

**Comment by lolodomo** [comment]
> Yes it's one binding consisting of several bundles. The binding.xml file is already translated in the org.openhab.binding.mqtt bundle.

OK

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**